test/helper.py

```diff
@@ -273,7 +293,7 @@ def _repr(v):
         write_string(info_dict_str.replace('\n', '\n        '), out=sys.stderr)
         self.assertFalse(
             missing_keys,
-            'Missing keys in test definition: {}'.format(', '.join(sorted(missing_keys))))
+            'Missing keys in test definition: {}'.format(', '.join(missing_keys)))
```

Comment:
While I'm not a fan of alphabetically sorting the **info dict** output, we could revert this to alphabetically sort the one-line error output:
```suggestion
            'Missing keys in test definition: {}'.format(', '.join(sorted(missing_keys))))
```